modules/dms-serverless-replication-config/README.md

```diff
@@ -0,0 +1,217 @@
+  # Cloud Posse recommends pinning every module to a specific version
+  # version     = "x.x.x"
+
+  endpoint_type                   = "source"
+  engine_name                     = "aurora-postgresql"
+  server_name                     = module.aurora_postgres_cluster.reader_endpoint
+  database_name                   = "postgres"
+  port                            = 5432
+  username                        = "admin_user"
+  password                        = "admin_password"
+  extra_connection_attributes     = ""
+  secrets_manager_access_role_arn = null
+  secrets_manager_arn             = null
+  ssl_mode                        = "none"
+
+  attributes = ["source"]
+  context    = module.this.context
+}
+
+# Upgrades to AWS DMS versions 3.4.7 and higher require that you configure AWS DMS to use VPC endpoints or use public routes.
+# This requirement applies to source and target endpoints for these data stores: S3, Kinesis, Secrets Manager, DynamoDB, Amazon Redshift, and OpenSearch Service.
+resource "aws_vpc_endpoint" "s3" {
+  vpc_endpoint_type = "Gateway"
+  vpc_id            = module.vpc.vpc_id
+  service_name      = "com.amazonaws.${var.region}.s3"
+  route_table_ids   = module.subnets.private_route_table_ids
+
+  tags = module.this.tags
+}
+
+module "s3_bucket" {
+  source = "cloudposse/s3-bucket/aws"
+  # Cloud Posse recommends pinning every module to a specific version
+  # version     = "x.x.x"
+
+  acl                          = "private"
+  versioning_enabled           = false
+  allow_encrypted_uploads_only = false
+  allow_ssl_requests_only      = false
+  force_destroy                = true
+  block_public_acls            = true
+  block_public_policy          = true
+  ignore_public_acls           = true
+  restrict_public_buckets      = true
+
+  context = module.this.context
+}
+
+module "dms_endpoint_s3_bucket" {
+  source = "cloudposse/dms/aws//modules/dms-endpoint"
+  # Cloud Posse recommends pinning every module to a specific version
+  # version     = "x.x.x"
+
+  endpoint_type = "target"
+  engine_name   = "s3"
+
+  s3_settings = {
+    bucket_name                      = module.s3_bucket.bucket_id
+    bucket_folder                    = null
+    cdc_inserts_only                 = false
+    csv_row_delimiter                = " "
+    csv_delimiter                    = ","
+    data_format                      = "parquet"
+    compression_type                 = "GZIP"
+    date_partition_delimiter         = "NONE"
+    date_partition_enabled           = true
+    date_partition_sequence          = "YYYYMMDD"
+    include_op_for_full_load         = true
+    parquet_timestamp_in_millisecond = true
+    timestamp_column_name            = "timestamp"
+    service_access_role_arn          = aws_iam_role.s3.arn
+  }
+
+  extra_connection_attributes = ""
+
+  attributes = ["target"]
+  context    = module.this.context
+}
+
+module "dms_replication_task" {
+  source = "cloudposse/dms/aws//modules/dms-replication-task"
```

Comment:
@oycyc was this meant to be serverless-replication-config?

```suggestion
  source = "cloudposse/dms/aws//modules/dms-serverless-replication-config"
```

Reply:
Yes great catch! I used the boilerplate and changed the code, but not the header 😅